doc/user/flycheck-versus-flymake.rst

```diff
@@ -225,9 +225,8 @@ run in sequence rather than concurrently.  For instance, Flycheck will check PHP
 files with PHP CLI first to find syntax errors, then with PHP MessDetector to
 additionally find idiomatic and semantic errors, and eventually with PHP
 CheckStyle to find stylistic errors.  The user will see all errors reported by
-all of these tools in the buffer.  However, if the first checker reported at
-least one error, then the subsequent checkers would not be run (by default; this
-behavior is configurable).
+all of these tools in the buffer.  This behavior is fully configurable; see
+:ref:`flycheck-checker-chains`.
```

Comment:
I think the 'this behavior' part is unclear with the previous sentence removed; don't you think?

Reply:
I agree.  I'm not sure why I removed it in the first place.